src/factory.js

```diff
@@ -326,6 +326,31 @@ const validateFSPIOPErrorCode = (code) => {
   }
 }
 
+/**
+ * Validate a code against the Mojaloop API spec, specifically custom errors, returns the incoming error code or throws an exception if invalid.
+ *
+ * @param code {number/string/object} - Mojaloop API spec error code (four digit integer as number or string or apiErrorCode enum)
+ * @param throwException {boolean} - Mojaloop API spec error code (four digit integer as number or string)
+ * @returns boolean -  if valid, true, if false then an exception will be thrown instead)
+ * @throws {FSPIOPError} - Internal Server Error indicating that the error code is invalid.
+ */
+const validateFSPIOPErrorGroups = (code) => {
+  const errorMessage = 'Validation failed due to error code being invalid'
+  let codeToValidate
+  if (typeof code === 'number' || typeof code === 'string') { // check to see if this is a normal error code represented by a number or string
```

Comment:
Error code should always be a string according to the API Spec, is it not?

Reply:
lets pursue this outside of this PR

Reply:
Hey Sam that is only for errors going out of the API that is a valid check, @mdebarros and @gibaros and myself had a chat abbout it